README.md

```diff
@@ -216,6 +216,87 @@ stripe.setAppInfo({
 
 This information is passed along when the library makes calls to the Stripe API.
 
+### Auto-pagination
+
+As of stripe-node 6.11.0, you may auto-paginate list api methods.
+We provide a few different API's for this to aid with a variety of node versions and styles.
+
+
+#### Async iterators (`for-await-of`)
+
+If you are in a node environment that has support for [async iteration](https://github.com/tc39/proposal-async-iteration#the-async-iteration-statement-for-await-of),
```

Comment:
Can we pluralize "Node" here like it is below?

Reply:
I assume you meant capitalize 😄 but yes, good catch!

Reply:
Haha, oops, yes.